.changeset/search-planets-flash.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-search': patch
+---
+
+Add `parentContext` prop to the `SearchContextProvider`, this added property does not create a local context and consumes the parent if it already exists.
```

Comment:
This changeset feels a little unclear from the perspective of a Backstage integrator reading through release notes.

Might be good to leave out the specifics of _what_ was changed (since it doesn't seem to change any behavior), but specify roughly _where_ it was changed, so that it's easier to pinpoint any problems, if there are any.

Maybe something like...

> Refactored `SearchModal` parent context logic.

Reply:
Great point, I will do it!

Reply:
Done 🙌🏻